web/vtadmin/src/components/routes/createWorkflow/CreateMoveTables.tsx

```diff
@@ -0,0 +1,324 @@
+                                value={formData.cells || ''}
+                            />
+                        </Label>
+                        <Select
+                            className="block grow min-w-[300px]"
+                            inputClassName="block w-full"
+                            items={onDDLOptions}
+                            label="OnDDL Strategy"
+                            onChange={(option) => setFormData({ ...formData, onDDL: option || '' })}
+                            placeholder={'Select the OnDDL strategy'}
+                            selectedItem={formData.onDDL}
+                        />
+                        <Select
+                            className="block grow min-w-[300px]"
+                            disabled={clustersQuery.isLoading}
+                            inputClassName="block w-full"
+                            itemToString={(cluster) => cluster?.name || ''}
+                            items={clusters}
+                            label="Cluster"
+                            onChange={(c) => setFormData({ ...formData, clusterID: c?.id || '' })}
+                            placeholder={clustersQuery.isLoading ? 'Loading clusters...' : 'Select a cluster'}
+                            renderItem={(c) => `${c?.name} (${c?.id})`}
+                            selectedItem={selectedCluster}
+                        />
+                        <MultiSelect
+                            className="block grow min-w-[300px]"
+                            inputClassName="block w-full"
+                            items={TABLET_OPTIONS}
+                            itemToString={(tt) => TABLET_TYPES[tt]}
+                            selectedItems={formData.tabletTypes}
+                            label="Tablet Types"
+                            onChange={(types) => setFormData({ ...formData, tabletTypes: types })}
+                            placeholder="Select tablet types"
+                        />
+                        <Label className="block grow min-w-[300px]" label="Source Time Zone">
+                            <TextInput
+                                onChange={(e) => setFormData({ ...formData, sourceTimeZone: e.target.value })}
+                                value={formData.sourceTimeZone || ''}
+                            />
+                        </Label>
+                    </div>
+
+                    <div className="my-2 mt-4">
+                        <div className="flex items-center">
+                            <Toggle
+                                className="mr-2"
+                                enabled={formData.autoStart}
+                                onChange={() => setFormData({ ...formData, autoStart: !formData.autoStart })}
+                            />
+                            <Label label="Auto Start" />
+                        </div>
+                        If enabled, the move will be started automatically.
```

Comment:
use `the workflow will be ...` instead of `the move will be` here in both `All Tables` and `Auto Start`